.rubocop.yml

```diff
@@ -0,0 +1,68 @@
+require:
+  - rubocop-rspec
+  - rubocop-performance
+
+AllCops:
+  DisabledByDefault: false
+  DisplayCopNames: true
+  TargetRubyVersion: 2.4
```

Comment:
as we're going to use 2.6 in production soon ( https://github.com/homeday-de/homeday/pull/1656 ), I think it makes sense to also target 2.6 here